src/overview/container.js

```diff
@@ -95,6 +129,34 @@ class OverviewContainer extends Component {
         return pills
     }
 
+    renderFilterPills(data, option) {
+        const Filterpills = data
+            .slice(0, 2)
```

Comment:
this `2` literal should be set to that new constant you added in the `constants` module. And camelCase ;)